projects/demo/src/modules/components/data-list/examples/1/index.html

```diff
@@ -1,8 +1,9 @@
+<!--TODO: TUI_ARROW migration-->
```

Comment:
Mention this TODO in this issue https://github.com/taiga-family/taiga-ui/issues/5808

Reply:
I will do it after I implement what to migrate to :)